go_lib/dependency/extenders/moduledependency/moduledependency.go

```diff
@@ -195,6 +195,12 @@ func (e *Extender) ValidateRelease(moduleName, moduleRelease string, version *se
 			}
 		}
 
+		// if parent version is empty, we think that module is not deployed
+		if parentVersion == "" {
+			validateErr = multierror.Append(validateErr, fmt.Errorf(`the "%s" module dependency "%s" is not deployed`, moduleName, parentModule))
```

Comment:
```
the "moduleA" module dependency "moduleB" does not meet the version constraint: the "moduleA" module dependency "moduleB" is not deployed
```

too much `moduleA` / `moduleB` words. Isn't it?

I guess the final error message expected to be like:
```
the "moduleA" module dependency "moduleB" does not meet the version constraint: "moduleB" is not deployed
```